source/security/authentication/aws-iam.txt

```diff
@@ -100,6 +100,28 @@ The driver checks for your credentials in the following sources in order:
 
          Omit the line containing ``AWS_SESSION_TOKEN`` if you don't need an AWS
          session token for that role.
+
+      AWS recommends using regional AWS STS endpoints instead of global 
+      endpoints to reduce latency, build-in redundancy, and increase session token validity.
+      To set the AWS region, set `AWS_REGION <https://docs.aws.amazon.com/sdkref/latest/guide/feature-region.html>`__ 
+      and `AWS_STS_REGIONAL_ENDPOINTS <https://docs.aws.amazon.com/sdkref/latest/guide/feature-sts-regionalized-endpoints.html>`__
+      as environment variables, as shown in the following example: 
+
+      .. code-block:: bash
+         
+         export AWS_STS_REGIONAL_ENDPOINTS=regional // Enables regional endpoints
+         export AWS_REGION=us-east-1 // Sets your AWS region
+         
+      If both these environment variables aren't set, the default region is 
+      ``us-east-1``. For a list of available AWS regions, see the
+      `Regional Endpoints <https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints>`__ 
+      section of the AWS Service Endpoints reference in the AWS documentation. 
+
+      .. warning:: Consult your SDK's Documentation for Setting an AWS Region
```

Comment:
I think we should remove or amend this warning: right now it's confusing because the driver uses aws-sdk/credential-providers; the warning is relevant to users who may not be using the node driver or who reuse their configuration elsewhere in their application

Reply:
Removed!